wgpu-core/Cargo.toml

```diff
@@ -19,6 +19,16 @@ trace = ["ron", "serde", "wgt/trace", "naga/serialize"]
 replay = ["serde", "wgt/replay", "naga/deserialize"]
 # Enable serializable compute/render passes, and bundle encoders.
 serial-pass = ["serde", "wgt/serde", "arrayvec/serde"]
+use-openxr = [
+    "openxr",
+    "hal/use-openxr",
+    "wgt/use-openxr",
+    "gfx-backend-vulkan/use-openxr",
```

Comment:
Current PR supports only Vulkan-based OpenXR (are others possible?)

Reply:
I think they are, at least dx and gl are supported by OpenXR.

Reply:
from what I've seen from the docs its GL and VK